src/resources/locales/en.json

```diff
@@ -284,7 +284,8 @@
             "pdfscalemode": "Page fit or width",
             "quitFullscreenTitle": "quit fullscreen mode",
             "readBookTitle": "start reading the book",
-            "settingsTitle": "settings"
+            "settingsTitle": "settings",
+            "historyPrevious": "history previous"
```

Comment:
better label: "Back in navigation history".

More generally, please be careful to have a capital on the first letter of each label.

Reply:
outdated, fixed since